doc/user/content/release-notes.md

```diff
@@ -57,7 +57,7 @@ Wrap your release notes at the 80 character mark.
   names.
 
   Materialize instances have a user named `materialize` installed, unless you
```

Comment:
I would say "automatically include a user named `materialize`" rather than "have a user installed".

Reply:
How would you suggest rewriting these given the clause that follows on the next line?

Reply:
Break it out into a separate sentence. The user's going to be installed no matter what, right? You have to drop it after creating the instance rather than being able to specify a "don't include" during creation.

Reply:
Sure––what is the edit you suggest?

Reply:
"Materialize instances automatically install a user named `materialize`. You can create additional users with [`CREATE USER`](/sql/create-user) and remove the default user with (`DROP USER`](\sql\drop-user)."